### PR TITLE
Improving selectors

### PR DESCRIPTION
**Pull request by Sebasgl23**
I have made a change in the first proceed to checkout button selecting it by his title because i think that when you click in add to cart button the only button that should appear with the text 'proceed to checkout' is it.

### CODE REVIEW

**Review by holgiosalos — CHANGES_REQUESTED** [review]
I see a lot of changes that are supposed to be already in main branch. Merge main branch with this branch to see the right changes as well as to solve the conflicts:


1. git pull origin main
2. git checkout improve-selectors
3. git merge main
4. Fix the conflicts and commit the result.
5. push the changes: git push -u origin improve-selectors

See [Resolving a merge conflict using the command line ](https://docs.github.com/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line#further-reading)for step-by-step instructions on resolving merge conflicts.

**Comment by Sebasgl23** [comment]
> I see a lot of changes that are supposed to be already in main branch. Merge main branch with this branch to see the right changes as well as to solve the conflicts:
> 
> 1. git pull origin main
> 2. git checkout improve-selectors
> 3. git merge main
> 4. Fix the conflicts and commit the result.
> 5. push the changes: git push -u origin improve-selectors
> 
> See [Resolving a merge conflict using the command line ](https://docs.github.com/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line#further-reading)for step-by-step instructions on resolving merge conflicts.

Thanks holmes, i was going crazy with all the errors, i think that i have fixed it with your suggestions

File: cypress/page/product-list.page.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ class ProductListPage {
 
   constructor() {
     this.addToCartButton = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
```
Comment (holgiosalos) [inline cypress/page/product-list.page.ts]: this selector also can be improved

**Review by holgiosalos — COMMENTED** [review]
Selectors on the following pages can be improved:

1. `menu-content.page.ts`
2. `shopping-cart.page.ts`

File: cypress/page/menu-content.page.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ class MenuContentPage {
 
 
   constructor() {
-    this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
+    this.tShirtMenu = "[title='T-shirts']";
```
Comment (holgiosalos) [inline cypress/page/menu-content.page.ts]: as we saw it in the last class, the appropriate selector should be:
`#block_top_menu > ul > li > a[title='T-shirts']`

So you don't need the call to the `eq` command if you use this selector

File: cypress/page/shopping-cart.page.ts [hunk 3]
```diff
@@ -4,11 +4,11 @@ class ShoppingCartPage {
   private proceedToCheckOutLogInButton: string;
 
   constructor() {
-    this.proceedToCheckOutLogInButton = ".cart_navigation span";
+    this.proceedToCheckOutLogInButton = "[title='Proceed to checkout']";
   }
 
   public proceedToCkeckoutLogin(): void {
-    cy.get(this.proceedToCheckOutLogInButton).click();
+    cy.get(this.proceedToCheckOutLogInButton).eq(1).click();
```
Comment (holgiosalos) [inline cypress/page/shopping-cart.page.ts]: you should not need the call to the `eq` command, if there are more than one HTML tag with the same title attribute, try to add something else to the selector. Example:
`.cart_navigation [title='Proceed to checkout']`

File: cypress/page/menu-content.page.ts (hunk 2 above)
Comment (Sebasgl23) [reply cypress/page/menu-content.page.ts]: Thanks holmes

File: cypress/page/shopping-cart.page.ts (hunk 3 above)
Comment (Sebasgl23) [reply cypress/page/shopping-cart.page.ts]: Thanks holmes

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**